examples/pylab_examples/demo_text_path.py

```diff
@@ -1,151 +0,0 @@
-
```

Comment:
BboxImage requires an axes, which this demo doesn't provide.  I actually think I'll just try to get this working again...